Library/Formula/ghc.rb

```diff
@@ -72,10 +74,12 @@ class Ghc < Formula
   def install
     ENV.m32 if build.build_32_bit?
 
-    # Build a static gmp (to avoid dynamic linking to ghc)
-    gmp_prefix = buildpath/"gmp-static"
+    # Build a static gmp (to avoid dynamic linking to ghc). "--with-pic" is
+    # *required* when building gmp statically, else you'll get
+    # "illegal text reloc" errors.
+    gmp_prefix = libexec/"gmp-static"
     resource("gmp").stage do
-      gmp_args = ["--prefix=#{gmp_prefix}", "--enable-cxx", "--enable-shared=no"]
+      gmp_args = ["--prefix=#{gmp_prefix}", "--disable-shared", "--with-pic"]
```

Comment:
Simply because it would never be used. [GHC uses only a few of GMP's primitives](https://ghc.haskell.org/trac/ghc/wiki/Design/IntegerGmp2#Link-timeinteger-libraryselection), and then implements the rest of the bignum functionality natively. (There's also an alternate package which eliminates GMP altogether, "integer-simple", but with a speed penalty, of course.) So presumably, if a hybrid C++/Haskell program needing bignum arithmetic (?) were to exist, it would bind against the Haskell library, not libgmpxx...